buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -196,14 +196,34 @@ def get_html_link_text(content, link):
 
 
 def print_flags_ready_to_flip(failed_jobs_per_flag, details_per_flag):
-    info_text = ["#### The following flags didn't break any passing jobs"]
+    info_text1 = ["#### The following flags didn't break any passing projects"]
     for flag in sorted(list(details_per_flag.keys())):
         if flag not in failed_jobs_per_flag:
             github_url = details_per_flag[flag].issue_url
-            info_text.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
-    if len(info_text) == 1:
-        return
-    print_info("flags_ready_to_flip", "success", info_text)
+            info_text1.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
+
+    if len(info_text1) == 1:
+        info_text1 = []
+
+    info_text2.append("#### The following flags didn't break any passing Bazel team owned/co-owned projects"):
+    for flag, jobs in failed_jobs_per_flag.items():
+        needs_bazel_team_migrate = False
+        for job in jobs:
+            pipeline, _ = get_pipeline_and_platform(job)
+            if bazelci.DOWNSTREAM_PROJECTS[pipeline].get("migration_owner") == "Bazel Team":
+                needs_bazel_team_migrate = True
+                break
```

Comment:
Maybe extract the for loop into a separate needs_bazel_team_migrate(jobs) function

Reply:
There should also be a second function for the "bazelci.DOWNSTREAM_PROJECTS[pipeline].get("migration_owner") == "Bazel Team"" part since that could be used below in line 289.

Reply:
On this line, we check if any of the given jobs needs to be migrated by the Bazel team.
On line 289, we just need to check if a single job, so it's not exact the same logic. 
But still, I think it's clear to extract the loop as a function, thanks!